apps/synapse-portal-framework/src/pages/FileEntityPage/SynapseFileEntityLinkCard.tsx

```diff
@@ -0,0 +1,46 @@
+import { Card, Link, Stack, Typography } from '@mui/material'
+import OpenInNewIcon from '@mui/icons-material/OpenInNew'
+import { ReactComponent as SynapseIconLogo } from './assets/SynapseIconLogo.svg'
+import { PRODUCTION_ENDPOINT_CONFIG } from 'synapse-react-client/utils/functions/getEndpoint'
+
+const SynapseFileEntityLinkCard = ({ synId }: { synId: string }) => {
+  const synapseUrl = `${PRODUCTION_ENDPOINT_CONFIG.PORTAL}Synapse:${synId}`
```

Comment:
We should really have a util to create a link to the Synapse entity page given an ID and optional version. Could you create one?